packages/legend-extension-dsl-persistence/src/graph-manager/__tests__/DSL_Persistence_Roundtrip.test.ts

```diff
@@ -70,4 +71,8 @@ test(unitTest('DSL Persistence V2 roundtrip'), async () => {
     TEST_DATA__roundtrip_non_temporal_snapshot_date_time_audit as Entity[],
     pluginManager,
   );
+  await TEST__checkBuildingElementsRoundtrip(
```

Comment:
this test is fine, but honestly, the better and more rigourous test to add is a full grammar roundtrip test in `packages/legend-manual-tests`, could you add one?

Reply:
Sure, have added one test case.